UI/MVUX/MVUX/MVUX/Presentation/FeedViewSample/FeedViewPage.xaml

```diff
@@ -0,0 +1,97 @@
+﻿<Page x:Class="MVUX.Presentation.FeedViewSample.FeedViewPage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:mvux="using:Uno.Extensions.Reactive.UI"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Grid utu:SafeArea.Insets="Bottom,Top">
```

Comment:
We should add a button here to re-kick off the calls so you can see the loading, custom loading and error messages appear again. Rght now, once it happens once you are just stuck with the end result